--- src/runtime/Config.x86-64-darwin.orig	2022-07-02 10:18:11.000000000 +0200
+++ src/runtime/Config.x86-64-darwin            2022-06-30 09:27:00.000000000 +0200
@@ -29,7 +29,8 @@
   OS_LIBS += -lpthread
 endif
 ifdef LISP_FEATURE_SB_CORE_COMPRESSION
-  OS_LIBS += -lzstd
+  OS_LIBS += -L%%%PREFIX%%%/lib -lzstd
+  CPPFLAGS += -I%%%PREFIX%%%/include
 endif
 ifdef LISP_FEATURE_SB_LINKABLE_RUNTIME
   LIBSBCL = libsbcl.a
